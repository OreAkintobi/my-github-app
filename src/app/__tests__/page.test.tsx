import React from 'react';
import { render } from '@testing-library/react';
import Page from '../page';

describe('Page', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Page />);
    expect(asFragment()).toMatchSnapshot();
  });
});
