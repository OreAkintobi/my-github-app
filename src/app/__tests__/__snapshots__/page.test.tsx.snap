// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="mx-auto min-h-lvh py-8 bg-gradient-to-r from-blue-100 via-blue-300 to-blue-500 text-white p-2"
  >
    <h1
      class="text-4xl text-black font-bold text-center mb-4"
    >
      GitHub User Search
    </h1>
    <div
      class="max-w-md mx-auto bg-white border-2 border-blue-500 rounded-lg p-6 shadow-lg"
    >
      <input
        class="w-full border border-gray-300 rounded-md px-4 py-2 mb-4 focus:outline-none focus:border-blue-500 text-black"
        placeholder="Enter GitHub username"
        type="text"
        value=""
      />
      <button
        class="w-full bg-blue-600 text-white rounded-md py-2 px-4 hover:bg-blue-700 focus:outline-none"
      >
        Search
      </button>
      <p />
      <p
        class="text-xl text-black font-bold text-center mt-4"
      >
        Switch Colors
      </p>
      <div
        class="flex justify-center mb-4"
      >
        <div
          class="w-12 h-12 rounded-full bg-blue-500 cursor-pointer mx-2 border-2 border-black"
        />
        <div
          class="w-12 h-12 rounded-full bg-green-500 cursor-pointer mx-2 "
        />
        <div
          class="w-12 h-12 rounded-full bg-purple-500 cursor-pointer mx-2 "
        />
        <div
          class="w-12 h-12 rounded-full bg-red-500 cursor-pointer mx-2 "
        />
      </div>
    </div>
    <div
      class="mt-8 max-w-md mx-auto transition-all"
    />
  </div>
</DocumentFragment>
`;
